File: Cpp/1550_Three_Consecutive_Odds.cpp
class Solution {
public:
    bool threeConsecutiveOdds(vector<int>& arr) {
        int odd_count = 0;
        for(auto& it : arr){
            if(it&1 == 1) ++odd_count;
            else odd_count = 0;
            if(odd_count == 3) return true;
        }
        
        return false;
    }
};

File: Kotlin/1550_Three_Consecutive_Odds.kt
class Solution {
    fun threeConsecutiveOdds(arr: IntArray): Boolean {
        var odd_count = 0
        arr.forEach{ it ->
            if(it and 1 == 1) ++odd_count
            else odd_count = 0
            
            if(odd_count == 3) return true
        }
        
        return false
    }
}